lib/songable.rb

```diff
@@ -0,0 +1,12 @@
+module Songable
+	require 'open-uri'
+  def list_query_results(query)
+  	puts "These are query results"
+  	query.gsub!(' ', '-')
+  	doc = Nokogiri::HTML(open("http://genius.com/search?q=#{query}"))
+
+		#I think this has same effect as #each_with_object([]) James?
```

Comment:
Lol actually it doesn't, each_with_object doesn't work in this context at all hahah, good move!  I added a map to it that produces hashes with :url and :text :+1: